src/components/yDragBoxLayer.ts

```diff
@@ -17,7 +17,7 @@ export module Components {
 
     public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
       super.computeLayout(origin, availableWidth, availableHeight);
-      this.bounds(this.bounds()); // set correct bounds when width/height changes
+      this._setBounds(this.bounds()); // set correct bounds when width/height changes
```

Comment:
Can we have the comment on a separate line to the code? Please? :D
 Add a line note